rewriter/Minitest.cc

```diff
@@ -174,7 +265,9 @@ unique_ptr<ast::Expression> runSingle(core::MutableContext ctx, ast::Send *send)
         ast::ClassDef::ANCESTORS_store ancestors;
         ancestors.emplace_back(ast::MK::Self(arg->loc));
         ast::ClassDef::RHS_store rhs;
-        rhs.emplace_back(prepareBody(ctx, std::move(send->block->body)));
+
+        unique_ptr<ast::Expression> body = std::move(send->block->body);
+        rhs.emplace_back(prepareBody(ctx, std::move(body), context));
```

Comment:
What's going on here? Why is this not one line like

```cpp
rhs.emplace_back(prepareBody(ctx, std::move(send->block->body), context));
```

Was there a C++ compiler error that was causing the code to need to be written this way?

Reply:
Oh, this was a bad artifact of a previous version of this PR that didn't get fully converted. (I'm realizing as I look at this feedback that I should have done a final once-over to catch a lot of these issues; sorry to have dumped this on you with all these issues still present!)